src/components/BastionSidebar.js

```diff
@@ -1,21 +1,30 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useLocation } from 'react-router-dom';
 import SidebarItems from './SidebarItems';
 import LinkHome from './LinkHome';
 
 const BastionSidebar = ({ bastion }) => {
-  const [active, setActive] = useState('Collections');
+  const pathname = useLocation().pathname;
+  let currentTab;
+  console.log(pathname, useLocation())
 
-  const changeActive = (title) => {
-    setActive(title);
-  };
+  if (pathname.includes('users')) {
+    currentTab = 'Users';
+  } else if (pathname.includes('cloudcode')) {
+    currentTab = 'CloudCode';
+  } else if (pathname.includes('files')) {
+    currentTab = 'Files';
+  } else {
+    currentTab = 'Collections';
+  }
 
   return (
     <div className='w-2/12 flex flex-col items-center bg-midnightblue py-12'>
       <LinkHome />
-      <SidebarItems active={active} setActive={changeActive} title={'Collections'} bastionName={bastion.StackName} />
-      <SidebarItems active={active} setActive={changeActive} title={'Users'} bastionName={bastion.StackName} />
-      <SidebarItems active={active} setActive={changeActive} title={'CloudCode'} bastionName={bastion.StackName} />
-      <SidebarItems active={active} setActive={changeActive} title={'Files'} bastionName={bastion.StackName} />
+      <SidebarItems active={currentTab} title={'Collections'} bastionName={bastion.StackName} />
+      <SidebarItems active={currentTab} title={'Users'} bastionName={bastion.StackName} />
+      <SidebarItems active={currentTab} title={'CloudCode'} bastionName={bastion.StackName} />
+      <SidebarItems active={currentTab} title={'Files'} bastionName={bastion.StackName} />
```

Comment:
changed it so that sidebar active state is determined by location.  This way it works if you refresh.  Might not be the most elegant solution, but should work.